src/components/App/App.js

```diff
@@ -1,20 +1,34 @@
 import './App.css';
 
 import React, { Component } from 'react';
+import styled from 'styled-components';
 import classNames from 'classnames';
 import MentorsList from '../MentorsList/MentorsList';
 import Filter from '../Filter/Filter';
-import Logo from '../Logo';
 import SocialLinks from '../SocialLinks/SocialLinks';
 import Header from '../Header/Header';
 import Modal from '../Modal/Modal';
 import ModalContent from '../Modal/ModalContent';
 import { toggle, get } from '../../favoriteManager';
 import { set } from '../../titleGenerator';
 import { report, reportPageView } from '../../ga';
-import MemberArea from '../MemberArea/MemberArea';
 import { getMentors } from '../../api';
 
+function scrollToTop() {
```

Comment:
What about `async function scrollToTop`?

We omit then the Promise construction and the resolve I think

Reply:
Out of scope, feel free to refactor anytime.

Reply:
👆